fbl31h/src/config/vehicles.json

```diff
@@ -1,29 +1,43 @@
+    { "make": "Aston Martin", "model": "DB4 GT Zagato", "topSpeed": 153.5, "year": 1960 },
+    { "make": "Ferrari", "model": "250 GTO", "topSpeed": 158, "year": 1962 },
+    { "make": "Studebaker", "model": "Avanti", "topSpeed": 168.2, "year": 1963 },
     { "make": "Iso", "model": "Grifo GL 365", "topSpeed": 161, "year": 1963 },
+    { "make": "Ferrari", "model": "500 Superfast", "topSpeed": 171, "year": 1964 },
     { "make": "AC", "model": "Cobra Mk III 427", "topSpeed": 165, "year": 1965 },
     { "make": "Lamborghini", "model": "Miura P400", "topSpeed": 171, "year": 1967 },
     { "make": "Ferrari", "model": "365 GTB/4 Daytona", "topSpeed": 174, "year": 1968 },
     { "make": "Lamborghini", "model": "Miura P400S", "topSpeed": 179.3, "year": 1969 },
+    { "make": "Ferrari", "model": "GT4 Berlinetta Boxer", "topSpeed": 175, "year": 1970 },
+    { "make": "Lamborghini", "model": "Countach LP400", "topSpeed": 179.0, "year": 1974 },
     { "make": "Lamborghini", "model": "Countach LP500 S", "topSpeed": 182, "year": 1982 },
     { "make": "Ruf", "model": "BTR", "topSpeed": 190, "year": 1983 },
+    { "make": "Ferrari", "model": "288 GTO", "topSpeed": 188.0, "year": 1984 },
     { "make": "Porsche", "model": "959", "topSpeed": 198, "year": 1986 },
     { "make": "Ruf", "model": "CTR", "topSpeed": 213, "year": 1987 },
+    { "make": "Ferrari", "model": "F40", "topSpeed": 202, "year": 1988 },
+    { "make": "Vector", "model": "W8", "topSpeed": 242, "year": 1990 },
+    { "make": "Bugatti", "model": "EB110 GT", "topSpeed": 209, "year": 1991 },
+    { "make": "Jaguar", "model": "XJ220", "topSpeed": 217.1, "year": 1992 },
     { "make": "McLaren", "model": "F1", "topSpeed": 221, "year": 1993 },
     { "make": "Bugatti", "model": "Veyron EB 16.4", "topSpeed": 253.81, "year": 2005 },
     { "make": "SSC", "model": "Ultimate Aero TT", "topSpeed": 256.18, "year": 2007 },
     { "make": "Bugatti", "model": "Veyron 16.4 Super Sport", "topSpeed": 267.856, "year": 2010 },
+    { "make": "Hennessey", "model": "Venom GT", "topSpeed": 270, "year": 2011 },
     { "make": "Koenigsegg", "model": "Agera RS", "topSpeed": 277.87, "year": 2017 },
-
-    { "make": "Stanley", "model": "Runabout", "topSpeed": 35, "year": 1890 },
-    { "make": "Mercedes", "model": "Simplex", "topSpeed": 60, "year": 1900 },
-    { "make": "Austro-Daimler", "model": "Prince Henry", "topSpeed": 85, "year": 1910 },
-    { "make": "Duesenberg", "model": "Model J", "topSpeed": 119, "year": 1920 },
-    { "make": "Duesenberg", "model": "Model SJ", "topSpeed": 140, "year": 1930 },
-    { "make": "Ferrari", "model": "GT4 Berlinetta Boxer", "topSpeed": 175, "year": 1970 },
-    { "make": "Ferrari", "model": "F40", "topSpeed": 202, "year": 1980 },
-    { "make": "Hennessey", "model": "Venom GT", "topSpeed": 270, "year": 2011 }
-]
\ No newline at end of file
+    { "make": "Bugatti", "model": "Chiron", "topSpeed": 304.7, "year": 2019 }  
```

Comment:
Source? Not in referenced article.

Reply:
I believe all of the cars not in the referenced article came from this article:

https://www.topgear.com/car-news/supercars/bugattis-300mph-chiron-vs-fastest-cars-history#1

I should have included it yesterday as a referenced article